{"name": "Negative test: Patch object with invalid data", "status": "skipped", "statusDetails": {"message": "Skipped: API does not support PATCH method (405)", "trace": "('C:\\\\Users\\\\Sergey\\\\Projects\\\\api-automation-framework\\\\tests\\\\test_patch_object_negative.py', 8, 'Skipped: API does not support PATCH method (405)')"}, "parameters": [{"name": "patch_data", "value": "{'text': None}"}], "start": 1748319259254, "stop": 1748319259254, "uuid": "312a6fdb-49c2-442f-9679-8d78ea0015b4", "historyId": "5fb6abfc7b49aa43a4142f0803622eba", "testCaseId": "70f805e9fced0522ce229ca93232757c", "fullName": "tests.test_patch_object_negative#test_patch_object_negative", "labels": [{"name": "tag", "value": "@pytest.mark.skip(reason='API does not support PATCH method (405)')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_patch_object_negative"}, {"name": "host", "value": "Sergo"}, {"name": "thread", "value": "8436-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_patch_object_negative"}]}